readme.md

```diff
@@ -110,17 +110,54 @@ type House struct {
     Words   string `json:"words"`
     Seats   []string `json:"seats"`
 }
-a := GoAxios{
+a := goaxios.GoAxios{
     Url:    "https://anapioficeandfire.com/api/houses/1",
     Method: "GET",
-    ResponseStruct: &House{},
```

Comment:
Hmm...the response struct should be a pointer to the struct so that interface assertion can work 🤔